bucket/openjdk20.json

```
@@ -0,0 +1,32 @@
+{
+    "description": "Official production-ready open-source builds of OpenJDK 20",
```

```suggestion
    "description": "Official Early-Access Builds of OpenJDK 20",
```
We can change the description to 'production-ready' once it becomes stable.